docs/7.0-API-Migration.md

```diff
@@ -34,4 +47,36 @@ Sidekiq.configure_server do |cfg|
 end
 ```
 
-Have more APIs which need a migration tip? Edit this file and open a PR!
\ No newline at end of file
+## Redis sentinels
+
+Due to Sidekiq's use of the new redis-client gem for connections, the configuration for it has changed.
+More details - https://github.com/redis-rb/redis-client#sentinel-support
+
+```ruby
+# broken - missing name keyword argument
+Sidekiq.configure_server do |cfg|
+  cfg.redis = {
+    url: "redis://myredis/0",
+    sentinels: [
+      {host: "host1", port: 26379},
+      {host: "host2", port: 26379},
+      {host: "host3", port: 26379}
+    ]
+  }
+end
+
+# fixed - don't forget to fix connection settings for both client and server
+Sidekiq.configure_server do |cfg|
+  cfg.redis = {
+    url: "redis://myredis/0",
```

Comment:
I don't think this line is correct since according to this https://github.com/sidekiq/sidekiq/blob/main/lib/sidekiq/redis_client_adapter.rb#L83C8-L83C8

```ruby
opts.delete(:url) if opts.key?(:sentinels)
```

if we have `:sentinels` key, the `:url` key will be dropped silently

Reply:
@ivanovv nice catch!
Looks like it's unexpected and should be removed. I'll look into this, thanks!